tests/python/gaia-ui-tests/gaiatest/tests/functional/homescreen/test_homescreen_delete_app_packaged.py

```diff
@@ -19,16 +19,11 @@ def setUp(self):
 
         self.test_data = {
             'name': 'packagedapp1',
-            'url': self.marionette.absolute_url('webapps/packaged1/manifest.webapp'),
+            'manifest_url': self.marionette.absolute_url('webapps/packaged1/manifest.webapp'),
             'title': 'Packaged app1'}
 
         # Install app so we can delete it
-        self.marionette.execute_script(
-            'navigator.mozApps.installPackage("%s")' % self.test_data['url'])
```

Comment:
Note that this uses installPackage, that's a different method than the normal webapps install. This is for installing a packaged app.